source/mainnet/net/index.rst

```diff
@@ -72,6 +72,7 @@ index
    :caption: Smart Contracts v1
 
    Introduction <../smart-contracts/general/introduction>
+   Best practices <../smart-contracts/best-practices/index>
```

Comment:
I don't think this should be right after the introduction. Perhaps after the on-chain guides but before Ethereum?